src/Symfony/Component/Console/Command/Command.php

```diff
@@ -241,6 +241,10 @@ public function run(InputInterface $input, OutputInterface $output)
             $this->interact($input, $output);
         }
 
+        if ($input->hasArgument('command') && null === $input->getArgument('command')) {
+            $input->setArgument('command', $this->getName());
```

Comment:
Writing this comment gave me another idea: is the command argument really a required argument? What if we made it optional instead of setting it here?